src/components/token/TokenMarkers.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from 'react';
+
+
+const TokenMarkers = (props) => {
+  const [token, setToken] = useState(props.token);
+
+  useEffect(() => {
+    setToken(props.token);
+  }, [props.token]);
+
+  const verifiedMarker = () => {
+    if (!token.verified) {
+      return null;
+    }
+
+    return (
+      <button className="info-hover-wrapper btn btn-link pl-2">
+        <i className="fa fa-check-circle fa-lg text-info" title="Verified"></i>
+        <span className="subtitle info-hover-popover">
+          This is a verified token. <a href="http://hathor.network" >Learn more.</a>
+        </span>
+      </button>
+    )
+  }
+
+  const bannedMarker = () => {
+    if (!token.banned) {
+      return null;
+    }
+
+    return (
+      <button className="info-hover-wrapper btn btn-link pl-2">
+        <i className="fa fa-exclamation-triangle fa-lg text-danger" title="Banned"></i>
+        <span className="subtitle info-hover-popover">
+          This token was banned and should not be trusted. <a href="http://hathor.network" >Learn more.</a>
```

Comment:
Same comment about the message and https in the URL

Reply:
this link (for banned) was removed